.gitignore

```diff
@@ -58,7 +58,7 @@ coverage.xml
 *.py,cover
 .hypothesis/
 .pytest_cache/
-
+tests/integration/aps/cassettes/
```

Comment:
why you ignore this 😅 ? This should be committed in order to run the tests with these cassettes

Reply:
we should remove it from here.